src/__tests__/views/loginView/__test__/loginView.test.js

```diff
@@ -0,0 +1,86 @@
+import React from "react";
+import { shallow, mount } from "enzyme";
+import { Provider } from "react-redux";
+import configureStore from "redux-mock-store";
+import { LoginView } from "../../../../views/loginView";
+
+const props = {
+  loginAction: jest.fn(),
+  onChangeRegister: jest.fn(),
+  history: {
+    push: val => jest.fn(val => val),
```

Comment:
You can have this as `push:  jest.fn()`,

Reply:
Thank you made changes